libraries/extractor/src/main/java/androidx/media3/extractor/DtsUtil.java

```diff
@@ -41,6 +41,9 @@ public final class DtsUtil {
   private static final int SYNC_VALUE_14B_BE = 0x1FFFE800;
   private static final int SYNC_VALUE_LE = 0xFE7F0180;
   private static final int SYNC_VALUE_14B_LE = 0xFF1F00E8;
+  private static final int SYNC_EXT_SUB_LE = 0x25205864;
```

Comment:
Do we have a specification pointing to these new values?

Reply:
1) Extension Substream Syncword (in different Endianness) are defined in Section 7.4.1:
https://www.etsi.org/deliver/etsi_ts/102100_102199/102114/01.06.01_60/ts_102114v010601p.pdf

2) The 2 FTOC Sync words (in different Endianness) are defined in Section 6.4.4.1 of
https://www.etsi.org/deliver/etsi_ts/103400_103499/103491/01.02.01_60/ts_103491v010201p.pdf

Reply:
@tianyif Do I need to add these as comments? Thanks.

Reply:
Yes, thanks!

Reply:
@tianyif Done. Please see 5c84b879f3837b289fd7af9c70eb55078adb705b. Thanks.